src/main/kotlin/lotto/model/WinNumber.kt

```diff
@@ -0,0 +1,43 @@
+package lotto.model
+
+/**
+ * 로또 당첨 번호 관리 클래스
+ * */
+class WinNumber private constructor(
+    private val _lastWinNumber: Lotto,
+    private val _bonusNumber: LottoNumber
```

Comment:
프로퍼티에는 언더바를 붙이지 않는 것이 좋습니다!
private setter public getter를 활용할 때 주로 사용합니다!
```suggestion
    private val lastWinNumber: Lotto,
    private val bonusNumber: LottoNumber
```